src/components/profile/AvatarUpload.tsx

```
@@ -0,0 +1,240 @@
+          tabIndex={disabled || isUploading ? -1 : 0}
+          aria-label="Upload profile picture"
+        >
+          <input
+            ref={fileInputRef}
+            type="file"
+            accept="image/*"
+            onChange={handleFileSelect}
+            className="hidden"
+            disabled={disabled || isUploading}
+            aria-hidden="true"
+          />
```

Biome (2.1.2)

[error] Disallow aria-hidden="true" from being set on focusable elements.

aria-hidden should not be set to true on focusable elements because this can lead to confusing behavior for screen reader users.
Unsafe fix: Remove the aria-hidden attribute from the element.


(lint/a11y/noAriaHiddenOnFocusable)